### PR TITLE
Feedback from in-house SEO guru

### PR DESCRIPTION
**Pull request by benarent**
Edits based on SEO minon, reducing duplication.

### CODE REVIEW

**Review by C-STYR — APPROVED** [review]

File: pages/ssh/index.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: What is SSH (Secure Shell Protocol)?
-description: SSH is a cryptographic network protocol for operating network services securely over an unsecured network. This post is a deep dive into what you can do with SSH.
+title: SSH
```
Comment (benarent) [inline pages/ssh/index.mdx:2]:
```suggestion
title: What is SSH (Secure Shell Protocol)?
```

File: pages/ssh/index.mdx [hunk 2]
```diff
@@ -111,6 +113,66 @@ import Section from "components/Section"
     </Flex>
   </Flex>
 
+    <H2 color="black" pt={4}>How to SSH?</H2>
+
+    SSH can be used to access any computer. While the primary use was to connect to servers, the definition of “compute” has become more
+    ubiquitous. For example, there are even very small SSH Servers, such as Dropbear SSH that only needs 110kb of space, designed for
+    “embedded”-type Unix systems such as routers and IoT devices. You can even [SSH into a self-driving vehicle](https://goteleport.com/blog/how-ssh-into-a-self-driving-vehicle/).
+
+    <H2 color="black" pt={4}>What is OpenSSH?</H2>
```
Comment (C-STYR) [inline pages/ssh/index.mdx:122]:
```suggestion
    <Box as="h2" color="black" pt={4}>What is OpenSSH?</Box>
```

Comment (C-STYR) [reply pages/ssh/index.mdx:122]: That's merely an example...

**Comment by C-STYR** [comment]
@benarent where we at with this? You ready for review again?

File: pages/ssh/index.mdx [hunk 3]
```diff
@@ -186,14 +188,14 @@ import Section from "components/Section"
...
+  <Box as="h2" id="ssh-production-best-practices" color="black" pt={4}>SSH production best practices</Box>  
     It’s a best practice to not directly connect to machines, but instead access them through a bastion or 
     [jump server](https://goteleport.com/blog/ssh-jump-server/). Bastions can be hardened machines that can be placed on the internet and can become the gateway to accessing the rest of your fleet.
     If you are looking for a quick guide on creating an SSH bastion, read our post on [setting up an SSH Bastion](https://goteleport.com/blog/ssh-bastion-host/) or our in-depth post on 
     [SSH bastion host best practices](https://goteleport.com/blog/security-hardening-ssh-bastion-best-practices/). Implementing 
     [certificate-based authentication is also considered the best method](https://goteleport.com/blog/comparing-passwordless-ssh-authentication-methods/) among password-based, key-based and host-based authentication. 
-</Section>
-
+  </Section>
+</Centrator>
```
Comment (benarent) [inline pages/ssh/index.mdx:198]:
```suggestion
```

File: pages/ssh/index.mdx [hunk 4]
```diff
@@ -238,8 +240,9 @@ import Section from "components/Section"
     src="../assets/tutorial.svg"
     smallIcon={true}
     caption="DEMO"
-    href="/resources/guides/using-teleport-with-openssh/"
+    href="/resources/guides/using-teleport-with-openssh//"
   >
     How to use Teleport with OpenSSH.
   </GridDisplay.Tile>
 </GridDisplay>
+<TryTeleport />
```
Comment (benarent) [inline pages/ssh/index.mdx:248]:
```suggestion
```

**Comment by benarent** [comment]
@C-STYR I've fixed the build errors, https://next-nkcawynjc-gravitational.vercel.app/ssh/ PTAL

**Pull request merged**